ui/desktop/README.md

```diff
@@ -176,6 +179,24 @@ Steps:
 
 TBD
 
+## Debug desktop client
+
+To debug the desktop client we recommend using node debugging client. 
+
+To start the electron app with the inspector adding a breakpoint before code starts:
+
+`$ yarn start:desktop --- --inspect-brk`.
```

Comment:
I've never seen `---`, only `--` what's that mean?

Reply:
@ZedLi yeah, the documentation shows just `--`. I am not sure why we use the `---` in front of the `--inspect` but if you do not use the `---` does not work.
Sorry I can not answer the question.

Reply:
Found the answer, it's for some reason `ember electron` specific, see [docs here](https://ember-electron.js.org/docs/guides/development-and-debugging#command-line-arguments).

Reply:
@ZedLi Awesome!!! :raised_hands: